pom.xml

```diff
@@ -1063,7 +1063,7 @@
             <dependency>
                 <groupId>com.microsoft.sqlserver</groupId>
                 <artifactId>mssql-jdbc</artifactId>
-                <version>7.0.0.jre8</version>
+                <version>8.4.1.jre8</version>
```

Comment:
is it the thing that gives LDT support?

Reply:
The first one with LDT was 7.2.1. @losipiuk suggested using the latest release. Actually, 9.2.0 has just been released (1 hour ago!) that's the first stable release for 9.x :-) https://github.com/microsoft/mssql-jdbc/releases/tag/v9.2.0

Reply:
Which one should we use? I went through the changelog and I don't see any new major features that would sound risky.

Reply:
Let's stay on 8.x (the stable stable) for now